mailpoet/lib/WooCommerce/MultichannelMarketing/MPMarketingChannel.php

```diff
@@ -41,8 +52,7 @@ public function get_description(): string { // phpcs:ignore PSR1.Methods.CamelCa
    * @return string
    */
   public function get_icon_url(): string { // phpcs:ignore PSR1.Methods.CamelCapsMethodName.NotCamelCaps
-    // TODO: use the correct image
-    return 'https://ps.w.org/mailpoet/assets/icon-256x256.png';
+    return $this->cdnAssetUrl->generateCdnUrl('icon-white-123x128.png');
```

Comment:
> This comment is about MPMarketingChannel itself. It seems kind of strange to me that it requires a URL while the installed extensions section can use an SVG. Any idea if we could somehow re-use that SVG instead?

It requires a [URL](https://github.com/woocommerce/woocommerce/blob/6fb787e954af29605d793c6cbb35c558d6e1d737/plugins/woocommerce/src/Admin/Marketing/MarketingChannelInterface.php#L42-L47C41), unfortunately.

I tested with an SVG, but it refused to load the image.

> One thing that's a little strange to me is that the design is different for the SVG and the icon, so the logo looks different when MailPoet shows up as a channel vs. an installed extension.

I noticed this as well, but since the design logo looks closer to our normal MP logo, I decided to go with that.

e.g.

![YsZdxM.png](https://github.com/mailpoet/mailpoet/assets/30554163/35bb74a5-a983-466f-9cd9-6a6198c99be3)

From: footer of https://www.mailpoet.com/